sources/packages/web/src/services/http/EducationProgramApi.ts

```diff
@@ -179,4 +181,40 @@ export class EducationProgramApi extends HttpBaseClient {
       throw error;
     }
   }
+
+  /**
+   * Ministry user approve's a pending program.
+   * @param programId program id.
+   * @param institutionId institution id.
+   * @param payload ApproveProgram.
+   */
+
+  public async approveProgram(
+    programId: number,
+    institutionId: number,
+    payload: ApproveProgram,
+  ): Promise<void> {
+    await this.patchCall(
+      `institution/education-program/${programId}/institution/${institutionId}/approve/aest`,
```

Comment:
It will be nice to mention the type of payload in method call to make it explicit like this.
![image](https://user-images.githubusercontent.com/54600590/153918827-01d4388c-fabd-435d-87c9-cf98e0ba603f.png)